rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/rename_class_with_same_name_but_different_namespace_multiple_use_statements.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureAutoImportNames;
+
+use Rector\Core\Rector\AbstractRector;
```

Comment:
What is this namespace for?

Reply:
Same as above. Gonna drop it. It is not the root cause.